### PR TITLE
fix cat encode

### PR DESCRIPTION
**Pull request by eromoe**

### CODE REVIEW

File: featurewiz/my_encoders.py [hunk 1]
```diff
@@ -56,7 +56,6 @@ def __init__(self):
     def fit(self,testx, y=None):
         ### Do not change this since Rare class combiner requires this test ##
         if isinstance(testx, tuple):
-            y = testx[1]
```
Comment (eromoe) [inline featurewiz/my_encoders.py:59]: No need in fit .

Comment (AutoViML) [reply featurewiz/my_encoders.py:59]: You have made a mistake in below code:
```
test_result = MLB.fit_transform(train[everycol])
```
Instead it should be:
```
test_result = MLB.transform(test[everycol])
```
Do you see the difference?

Comment (eromoe) [reply featurewiz/my_encoders.py:59]: Right , my mistake , now fixed

**Comment by eromoe** [comment]
@AutoViML 
I haven't look into why rare class need label encoder return tuple ,  could you mind to make an explanation  ? 

If , tuple return is only used for something relate to rare class . Don't you think two label_encoder are more correct ?
Such as 
1. One only fit one string array and return one  transformed array
2. The other fit (string array, y ) and return (transformed array, y )

**Comment by AutoViML** [comment]
At the moment let's close this thread with this change. If you feel another change is needed elsewhere, you can open another pull request. Okay?

**Comment by eromoe** [comment]
OK, I won't make pr in recently I think ,  just gona make some discussion  :)
